src/ontology/uberon-edit.obo

```diff
@@ -222532,6 +222532,16 @@ synonym: "sagittal ridge" EXACT []
 is_a: UBERON:4200133 ! crest
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500117
+name: opisthocranion
+def: "A craniometric point on the nuchodorsal border of a foramen magnum in the medial plane." [ISBN:9780873659505, Wiktionary:opisthocranion]
```

Comment:
Hi, @megbalk, the wiktionary definition I see is:

"([anatomy](https://en.wiktionary.org/wiki/anatomy)) The [posteriormost](https://en.wiktionary.org/wiki/posteriormost) point in the [midsagittal](https://en.wiktionary.org/wiki/midsagittal) plane of the [occiput](https://en.wiktionary.org/wiki/occiput)"
This definition aligns with the images (of human skulls) I see with a cursory online search.

I am not familiar with the term "nuchodorsal" and could not find a definition online. Even so, this point does not appear to be part of the foramen magnum, at least not in humans.

Reply:
@bvarner-ebi thank you for checking. The definition comes from von den Driesch 1979. I'll find other definitions, then, to make sure that it is more broad and applicable.

Reply:
The definition [here](https://locatelambda.org/manual/) has the opisthion and the opisthocranion, which point is on the skin, not a bone (but paleontologists define it on the skull). 

I will change this to be the opposite side from the glabella on the skull.

I must've gotten this confused with the opisthion, and so would also like to make that term.

Reply:
Hi, @megbalk,
To consider: add mention that the point exists in the midline since that appears in all the definitions referenced that I can see. 
Please also add dbxref (probably just the URL) that supports this text definition.